pkg/volume/cephfs/cephfs_test.go

```diff
@@ -27,8 +29,13 @@ import (
 )
 
 func TestCanSupport(t *testing.T) {
+	tmpDir, err := ioutil.TempDir("/tmp", "fake")
+	if err != nil {
+		t.Fatalf("can't make a temp dir")
```

Comment:
You may want to include the error in the ```Fatalf``` as it may contain information as to why the directory could not be created.